pkg/manifest/manifest_test.go

```diff
@@ -1,113 +0,0 @@
-//go:build unit
-
-// @license
-// Copyright 2021 Dynatrace LLC
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-package manifest
-
-import (
-	"github.com/google/go-cmp/cmp/cmpopts"
-	"gotest.tools/assert"
-	"testing"
-)
-
-var sortStrings = cmpopts.SortSlices(func(a, b string) bool { return a < b })
-
-func TestManifestFilterEnvironmentsByNamesWithEmptyNames(t *testing.T) {
```

Comment:
you've covered all of these cases with the ones you added for the filtering on load right?

Reply:
Yes